mirage/factories/collection-provider.ts

```diff
@@ -26,3 +31,16 @@ export default Factory.extend<CollectionProvider>({
         return `${faker.lorem.paragraph()} Find out <a href="https://help.osf.io/">more</a>.`;
     },
 });
+
+declare module 'ember-cli-mirage/types/registries/schema' {
+    export default interface MirageSchemaRegistry {
+        collectionProviders: MirageCollectionProvider;
+    } // eslint-disable-line semi
```

Comment:
Also, are we able to remove the eslint-disable? It may cause tests to fail - is there a good reason to keep it?